### ISSUE TITLE
ref dom time issue

### ISSUE THREAD

**Issue by ernieyang09** [react-component/tabs#128]
using correct in dev mode.

Build successful and run fail in production mode.

TypeError: Cannot read property 'ownerDocument' of undefined

I trace the code and find interesting things

```
export default class InkTabBarNode extends React.Component {
  componentDidMount() {
    if (isDev) {
      // https://github.com/ant-design/ant-design/issues/8678
      this.timeout = setTimeout(() => {
        componentDidUpdate(this, true);
      }, 0);
    } else {
      componentDidUpdate(this, true);
    }
  }
...
```

I found that componentDidUpdate function can't find ref rootNode.

and I edit code

```
 componentDidMount() {
      // https://github.com/ant-design/ant-design/issues/8678
      this.timeout = setTimeout(() => {
        componentDidUpdate(this, true);
      }, 0);
  }
```

it works

I print in lifecycle and found componentDidUpdate function will exec before ref rootNode

and the solution is put componentDidUpdate in next tick

I'm using react 0.16. Don't know if this issue related to version.

Maybe It should use some new ref api in 16version?

**Comment by ernieyang09** [comment]
I downgrade from v9.3.0 to 9.2.5 and it seems work well

**Comment by picodoth** [comment]
fixed in 9.3.4
